project/src/main/java/com/google/autograder/servlets/GetCourseDetailsServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.google.autograder.servlets;
+
+import java.net.URL;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Entity;
+import com.google.autograder.servlets.helpers.Services;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+
+@WebServlet("/getCourseDetails")
```

Comment:
Do these new servlets need a mapping in web.xml?

Reply:
Yes, I add servlet mapping in a later PR.